src/Plot.js

```diff
@@ -570,12 +577,12 @@ export default class Plot extends Viz {
 
     const y2Transform = `translate(-${this._margin.right}, ${this._margin.top + topOffset})`;
     const y2Group = elem("g.d3plus-plot-y2-axis", {parent, transition, enter: {transform: y2Transform}, update: {transform: y2Transform}});
-
+    const timeMargin = (xTime || x2Time) && y2Exists ? y2TestRange[0] : 0;
```

Comment:
This solution does not work for all test cases, and is a band-aid over the real problem, which I believe is related to how the `xDifference` value is being calculated. Take a look at this example code, which does not use any secondary axes but breaks when using the `time` method:

```js
var data = [
    {id: "alpha", alphaX: 2004, alphaY:  7},
    {id: "alpha", alphaX: 2005, alphaY: 25},
    {id: "alpha", alphaX: 2006, alphaY: 13},
];

new d3plus.LinePlot()
  .data(data)
  .groupBy("id")
  .x("alphaX")
  .y("alphaY")
  .time("alphaX") // comment this line to see the difference
  .timeline(false)
  .render();
```